### PR TITLE
Buff slow healer limb regen so it is in line with other healing traits and better describe healing traits.

### PR DESCRIPTION
**Pull request by meladath**
#### Summary

SUMMARY: Balance "Buff SLOWHEALER mending_modifier so it is in line with other healing traits, also better descriptions for healing traits"

#### Purpose of change

Considering the points cost of slow healer, as well as the fact that the healing system has changed considerably since these traits were implemented, I figure this can be rebalanced to bring it in line with the other healing traits.

I also made sure to update the descriptions of all healing traits, so that they correctly tell the player what they do.

#### Describe the solution

Slow healer now causes you to heal your limbs at 75% the normal rate. This brings it in line with the other healing traits which provide the same debuff to HP and limb healing.

#### Testing

1. Get hit
2. Get broken limb
3. Heal!

### CODE REVIEW

**Comment by Fris0uman** [comment]
Why does 75% makes more sense than 50%?

**Comment by meladath** [comment]
> 
> 
> Why does 75% make more sense than 50%?

A good question. I think this because both imperceptive healer and poor healer both reduce your sleeping and awake healing by 90% and 66% respectively. Slow healer on the other hand reduces your sleeping healing by 25%, however it reduces your awake healing by 50%, not 25%. Makes it an anomaly when you compare it to the other negative traits.

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -148,7 +148,7 @@
     "id": "FASTHEALER",
     "name": { "str": "Fast Healer" },
     "points": 2,
-    "description": "You heal faster when sleeping and will even recover a small amount of HP when not sleeping.",
+    "description": "Your wounds heal themselves quicker than usual. You heal 50% faster whilst asleep and heal at 20% of that rate whilst awake. Your broken limbs also heal twice as fast.",
```
Comment (Night-Pryanik) [inline data/json/mutations/mutations.json:151]: Double spaces after periods.

File: data/json/mutations/mutations.json [hunk 2]
```diff
@@ -774,18 +774,18 @@
     "id": "SLOWHEALER",
     "name": { "str": "Slow Healer" },
     "points": -2,
-    "description": "You heal a little slower than most; sleeping will heal less HP.",
+    "description": "Your wounds heal a little slower than most, your HP and broken limbs recover at 75% of the normal rate",
```
Comment (Night-Pryanik) [inline data/json/mutations/mutations.json:777]: Period in the end.

File: data/json/mutations/mutations.json [hunk 3]
```diff
@@ -798,7 +798,7 @@
     "id": "SLOWHEALER3",
     "name": { "str": "Imperceptive Healer" },
     "points": -8,
-    "description": "You recover barely any health through sleeping - it will heal only one tenth of usual HP.",
+    "description": "Wounds are incredibly dangerous to you, as they barely heal at all. Your broken limbs and HP only recover at 10% of the regular rate.",
```
Comment (Night-Pryanik) [inline data/json/mutations/mutations.json:801]: Double spaces after periods.

File: data/json/mutations/mutations.json [hunk 4]
```diff
@@ -1544,7 +1544,7 @@
     "id": "FASTHEALER2",
     "name": { "str": "Very Fast Healer" },
     "points": 4,
-    "description": "Your flesh regenerates slowly, and you will regain HP even when not sleeping.",
+    "description": "Your wounds heal very quickly. You heal 50% faster whilst asleep and heal at 66% of that rate whilst awake. Your broken limbs also heal 4 times faster than usual.",
```
Comment (Night-Pryanik) [inline data/json/mutations/mutations.json:1547]: Double spaces after periods.

File: data/json/mutations/mutations.json [hunk 5]
```diff
@@ -1558,7 +1558,7 @@
     "id": "REGEN",
     "name": { "str": "Regeneration" },
     "points": 6,
-    "description": "Your flesh regenerates from wounds incredibly quickly.",
+    "description": "Your flesh regenerates from wounds incredibly quickly. You heal 150% faster whilst asleep and heal at 200% of that rate whilst awake. Your broken limbs also heal 16 times faster than usual.",
```
Comment (Night-Pryanik) [inline data/json/mutations/mutations.json:1561]: Double spaces after periods.

File: data/json/mutations/mutations.json [hunk 6]
```diff
@@ -1573,7 +1573,7 @@
     "points": 5,
     "valid": false,
     "purifiable": false,
-    "description": "Your broken limbs mend themselves without significant difficulty.",
+    "description": "Your broken limbs mend themselves without significant difficulty. You do not require splints and broken limbs heal 20 times faster than usual.",
```
Comment (Night-Pryanik) [inline data/json/mutations/mutations.json:1576]: Double spaces after periods.

**Pull request merged**